build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/AnalyzeArtifactSizeMetrics.kt

```diff
@@ -109,34 +110,73 @@ internal abstract class AnalyzeArtifactSizeMetrics : DefaultTask() {
         return ArtifactSizeMetricsAnalysis(artifactSizeMetrics, significantChange, changeHappened)
     }
 
+    // There are small fluctuations in artifact size that are not real delta
+    private fun Long.isNotaFluctuation() = abs(this) > 5L
```

Comment:
**question:** Is this number big enough? In your example comment, there was a delta of 10 bytes that also seems like a fluctuation.

Reply:
I'm basing it off the fluctuations seen here: https://github.com/awslabs/aws-sdk-kotlin/pull/1291. They're all less than 5. That 10 I added manually but I'm not opposed to upping the threshold

Reply:
Ah ok, that seems fine then. I didn't realize the 10 was a manual data point

Reply:
If we're hiding the unimportant/no-action-required changes by default, do we still need to filter out these fluctuations?

Reply:
It's not really necessary but I also wouldn't consider it real delta

Reply:
It pollutes an already large table